source/account_structure/index.html.md.erb

```diff
@@ -0,0 +1,51 @@
+---
+title: Account structure 
+weight: 55
+---
+
+# Account structure 
+
+If you have multiple services integrated, you can treat them separately or as
+a combined single service. You can use the GOV.UK Pay platform to set this this within
+GOV.UK Pay and within your Payment Service Provider (PSP). 
+
+![Account structures diagram](https://raw.githubusercontent.com/alphagov/pay-tech-docs/account_structure/source/images/account_structure.png)
+
+You may have 2 services, for example for parking permits and parking fines.
+The diagram takes this example and illustrates the different possible
+structures you can have:
+
+### Type 1 (GOV.UK Pay separate, PSP separate)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+both GOV.UK Pay and the PSP.
+
+### Type 2 (GOV.UK Pay combined, PSP combined)
+
+"Parking permits" and "Parking fines" are considered as a single combined
+service in GOV.UK Pay and in the PSP.
+
+### Type 3 (GOV.UK Pay separate, PSP combined)
+
+"Parking permits" and "Parking fines" are considered as separate services in
+GOV.UK Pay and as a single combined service in the PSP. 
```

Comment:
Destination bank account will be the same for both services.

Reply:
Done 👍 and for Type 2